lib/cacheServerClient.ts

```diff
@@ -166,13 +179,58 @@ export class CacheServerClient {
     namespace,
   }: {
     namespace: string;
-  }): Promise<IRoleDefinition> {
+  }): Promise<IRoleDefinitionV2> {
     const { data } = await this.httpClient.get<IRole>(`/role/${namespace}`);
     return data.definition;
   }
 
-  async getOffchainClaims({ did }: { did: string }): Promise<OffchainClaim[]> {
-    const { data } = await this.httpClient.get<{ service: OffchainClaim[] }>(
+  async getRoleEIP191JwtsBySubject(
+    subjectDID: string,
+    { isAccepted, namespace }: CredentialFilters = {}
+  ): Promise<RoleEIP191JWT[]> {
+    const { data } = await this.httpClient.get<Credential[]>(
+      `/claim/subject/${subjectDID}`,
+      {
+        params: {
+          isAccepted,
+          namespace,
+        },
+      }
+    );
+    let roleEIP191Jwts;
+    data.map((credential) => {
+      if (credential.issuedToken) {
+        roleEIP191Jwts.push({
+          payload: jwt.decode(credential.issuedToken) as RolePayload,
+          eip191Jwt: credential.issuedToken,
+        });
+      }
+    });
+    return roleEIP191Jwts;
+  }
+
+  async getVerifiableCredentialBySubject(
+    subjectDID: string,
+    { isAccepted, namespace }: CredentialFilters = {}
+  ): Promise<VerifiableCredential<RoleCredentialSubject>[]> {
+    const { data } = await this.httpClient.get<Credential[]>(
+      `/claim/subject/${subjectDID}`,
+      {
+        params: {
+          isAccepted,
+          namespace,
+        },
+      }
+    );
+    let vcs;
+    data.map((credential) => {
+      vcs.push(credential.vp.verifiableCredential);
+    });
+    return vcs;
+  }
```

Comment:
Can this code be removed as I don't think we're using the `claims` interface anymore?